.stylelintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "extends": "stylelint-config-standard"
```

Comment:
We will also want `stylelint-config-recommended-scss`, since that will have rules that are specific to SCSS/

Reply:
Actually, I think `stylelint-config-standard-scss` is probably a better fit. I'll updated the card to reflect that.